cmds/ocm/commands/ocmcmds/common/inputs/types/ociartifact/type.go

```diff
@@ -1,12 +1,13 @@
 package ociartifact
 
 import (
+	"ocm.software/ocm/api/ocm/extensions/artifacttypes"
 	"ocm.software/ocm/cmds/ocm/commands/ocmcmds/common/inputs"
 )
 
 const (
-	TYPE        = "ociArtifact"
-	LEGACY_TYPE = "ociImage"
+	TYPE        = artifacttypes.OCI_ARTIFACT
+	LEGACY_TYPE = artifacttypes.OCI_IMAGE
```

Comment:
@open-component-model/maintainers - can't we simply drop the LEGACY part?

Reply:
I would also like to see OCI Image completely removed (from the spec) as I don't see any value in keeping it. And yes, whoever uses it, will need to correct his CVs...

Reply:
@morri-son - I will not do that in this PR.
@fabianburth might create a poll for our naming convention. When we all agree on the names and have a ticket for the same, it will require quite some refactoring, which I'm not doing now in this PR.

@morri-son - if you're ok with the approach, please resolve the conversation